wicket/aan-de-slag.md

```diff
@@ -0,0 +1,7 @@
+# Aan de slag als developer
+
+Je kunt op dit moment de laatste `.jar` vinden als download bij [GitHub Actions voor Apache Wicket bij de `main` branch](https://github.com/nl-design-system/example/blob/main/.github/workflows/continuous-deployment.yml#L166-L179).
```

Comment:
Dit is voor Wicket een onconventionele manier van werken. Om een jar als maven dependency op te nemen, kan je SNAPSHOTs publiceren naar een Maven repository; dit moet vanuit Maven (of Gradle). Uiteraard kan pnpm wel Maven triggeren. Zie https://central.sonatype.org/publish/publish-maven/#performing-a-snapshot-deployment voor hoe we dit kunnen doen.

Een andere manier om het aan Java devs beschikbaar te krijgen, die nog geen Sonatype snapshot deployment vereist:

* git clone  de  nl-design-system rotterdam git repository
* Voor `mvn clean install uit` met Java 21
* Voeg een dependency toe aan je project.

Dependency voor nl design system:

```xml
        <dependency>
            <groupId>nl.rotterdam.nl-design-system</groupId>
            <artifactId>components-wicket</artifactId>
            <version>1.0-SNAPSHOT</version>
        </dependency>
```

Reply:
Wellicht snel https://github.com/nl-design-system/rotterdam/issues/128 oppakken?